app/views/users/_blocked_person.haml

```diff
@@ -0,0 +1,12 @@
+.media.blocked_person{id: person.id}
+  .pull-right
+    = link_to t("users.privacy_settings.stop_ignoring"), block_path(block),  class: "btn btn-danger", method: :delete
+  .media-object.pull-left
+    = person_image_link(person)
```

Comment:
The [people search page](https://github.com/diaspora/diaspora/blob/develop/app/views/people/index.html.haml#L36) uses `people/person` [which is correct](https://github.com/diaspora/diaspora/blob/develop/app/views/people/_person.html.haml#L7). Did you adapt your code from some other partial? In that case: sure, why not. (But, of course, you don't have to do it.)